pygsp/graphs/nngraphs/cube.py

```diff
@@ -35,20 +35,23 @@ class Cube(NNGraph):
     """
 
     def __init__(self,
-                 radius=1,
                  nb_pts=300,
                  nb_dim=3,
+                 length=1,
                  sampling='random',
                  seed=None,
                  **kwargs):
 
-        self.radius = radius
         self.nb_pts = nb_pts
         self.nb_dim = nb_dim
+        self.length = length
         self.sampling = sampling
         self.seed = seed
         rs = np.random.RandomState(seed)
 
+        if length != 1:
+            raise NotImplementedError('Only length=1 is implemented.')
+
```

Comment:
I think you can remove this exception if you just multiply `pts` by length.

Reply:
It's actually mostly pointless to scale the embedding. The same effect can be achieved by reducing the kernel width. I removed this option in `sphere-graphs`.